### PR TITLE
Issue 110 - Add Test Coverage to Query Components

### PR DESCRIPTION
**Pull request by cdanwards**
Fixes 110

## Creator

- [x] Build relevant tests if any apply
- [x] Ensure there are no new warnings (in app AND in storybook)
- [x] Upload an animated GIF showcasing the solution (if it applies)
- [x] Set a relevant reviewer
- [ ] Add a line to the changelog documenting the high-level change this PR introduces

## Reviewer

- [ ] Review CI build results
- [ ] Review function and form on iOS
- [ ] Review function and form on Android
- [ ] Review new stories if any apply
- [ ] Review test coverage: Are all new or changed states tested?
- [ ] Review new test logic
- [ ] Review the changelog entry

### CODE REVIEW

File: mobile/src/live/LiveButton.tests.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    },
+  },
+];
+
+it('renders without error', () => {
+  renderer.create(
+    <Providers mocks={mocks} addTypename={false}>
+      <LiveNowButton isLiveNow />
+    </Providers>
+  );
```
Comment (conrad-vanl) [inline mobile/src/live/LiveButton.tests.js]: Is this missing a snapshot check, like the others?

```
const tree = renderer.create(...);
expect(tree).toMatchSnapshot();
```

**Comment by IsaacHardy** [comment]
@cdanwards What are we waiting on with this PR? Have you had time to answer Conrad, above?

File: mobile/src/live/LiveButton.tests.js (hunk 1 above)
Comment (cdanwards) [reply mobile/src/live/LiveButton.tests.js]: That's correct. I was following a blog tutorial thing for this one and they weren't using snapshot. I'll get it in there!

**Comment by cdanwards** [comment]
@IsaacHardy I made the recommended changes. So this should be good to go.

**Comment by conrad-vanl** [comment]
looks like there are snapshot failures now

**Pull request merged**